Minor fix for mpd

anxdpanic opened the pull request:
params adjustment
- adjust params, keep all possible params while still able to trigger mpd video return
- add eurl param to avoid some restrictions

anxdpanic committed:
add .gitignore

anxdpanic committed:
params adjustment

- adjust params, keep all possible params while still able to trigger mpd video return
- add eurl param to avoid some restrictions

h0d3nt3uf3l commented:
Thanks for your push.
May I ask what are you working on? Think we have to talk to each other so that the work won't get done twice.
Saw you're programmed another switching method, seems like more professional than mine.

I think about not to use inputstream.mpd and give ffmpeg a try, because inputstream.mpd don't have all codecs available. Get a few time an error that this or that coded is not implemented.
If I have time in about 6 hours I'll push my changes.

Whats the param adjustment for?

anxdpanic commented:
The param adjustment just sends as much in both types (mpd or usual) to 
get the video info, the eurl param stops some 'this video can't be 
viewed from this site' type errors that I ran into

Edit: removed email reference
Edit: removed anything non relevant to PR

anxdpanic commented:
Closing this, I submitted full PR for mpeg-dash to Kolifanes/plugin.video.youtube :+1:
